### PR TITLE
CONN-1076: Mockito Junit

### PR DESCRIPTION
**Pull request by sailajaa**

### CODE REVIEW

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/CreateuserBean.java [hunk 1]
```diff
@@ -82,14 +85,12 @@ public String addCreateUser() {
...
-        if (this.getUserName() != null && this.getPassword() != null) {
-        }
         Login user = new Login(userName, password);
         try {
             createdUser = loginService.addUser(user);
-            if (createdUser) {
-                FacesContext facesContext = FacesContext.getCurrentInstance();
-                HttpSession session = (HttpSession) FacesContext.getCurrentInstance().getExternalContext().getSession(false);
+            if (createdUser) {                
+                //HttpSession session = (HttpSession) FacesContext.getCurrentInstance().getExternalContext().getSession(false);
```
Comment (msweaver) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/CreateuserBean.java]: need to remove dead code.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/CreateuserBean.java [hunk 2]
```diff
@@ -27,7 +27,6 @@
  *    */
```
Comment (msweaver) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/CreateuserBean.java]: this file needs formatting and javadocs.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/test/java/gov/hhs/fha/nhinc/admingui/managed/CreateuserBeanTest.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
+/*
+ *   /*
```
Comment (msweaver) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/test/java/gov/hhs/fha/nhinc/admingui/managed/CreateuserBeanTest.java]: Seems like there are double comments here. Can this be cleaned up?

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/hibernate/LoginServiceImpl.java [hunk 4]
```diff
@@ -48,16 +47,20 @@
 @Service
 public class LoginServiceImpl implements LoginService {
 
-    private static Logger log = Logger.getLogger(LoginServiceImpl.class);
+    private static final Logger log = Logger.getLogger(LoginServiceImpl.class);
 
     @Autowired
     private UserLoginDAO userLoginDAO;
 
+    LoginServiceImpl(UserLoginDAO mockDao) {
```
Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/hibernate/LoginServiceImpl.java]: I would change the name of this input as well.  change it to userLoginDao.

**Pull request merged**